infrastructure/kube/keep-test/geth-node/eth-goerli-node.yaml

```diff
@@ -44,7 +44,7 @@ spec:
     spec:
       containers:
         - name: geth-goerli
-          image: ethereum/client-go:v1.10.20
+          image: ethereum/client-go:v1.10.24
```

Comment:
@nkuba Do we need to run Prysm for testnet?

Reply:
Yes we do need to run it, see https://github.com/keep-network/keep-core/issues/3264#issuecomment-1246591600